dm/syncer/syncer.go

```diff
@@ -413,6 +418,12 @@ func (s *Syncer) Init(ctx context.Context) (err error) {
 			}
 		}
 	}
+	s.flushCpWorker = &checkpointFlushWorker{
+		input:        nil, // will be created in s.reset()
```

Comment:
```suggestion
		input:        nil, // will be created in s.newJobChans()
```
btw, how about move the logic of `flushCpWorker ` to another file? `syncer.go` is already too large 😂